### PR TITLE
Finished HW React1-week2

### PR DESCRIPTION
**Pull request by RodrigoGuinsbergPinto**

### CODE REVIEW

File: react/react1/week2/app/src/components/Header.jsx [hunk 1]
```diff
@@ -0,0 +1,7 @@
+function Header(props) {
+  const { title } = props;
```
Comment (shpomp) [inline react/react1/week2/app/src/components/Header.jsx:2]: Nice to use destructuring 👍🏽

File: react/react1/week2/app/src/components/todosData.jsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  },
+  {
+    id: 2,
+    description: "Brush teeth",
+  },
+  {
+    id: 3,
+    description: "Eat breakfast",
+  },
+];
```
Comment (shpomp) [inline react/react1/week2/app/src/components/todosData.jsx:14]: Nitpick, but this would not hold any JSX, so it could (should?) be just a .js file 👍🏽

**Review by shpomp — APPROVED** [review]
Very nice, clean and clear 🤩 
Nothing much to even add 🤷🏽 
Keep up the learning 👍🏽  ⭐

File: react/react1/week2/app/src/components/todosData.jsx (hunk 2 above)
Comment (RodrigoGuinsbergPinto) [reply react/react1/week2/app/src/components/todosData.jsx:14]: Hi Karo,
thanks for reviewing my hw.
Regarding the .jsx file, I thought we should always use it in React.
I'll read about .js vs. .jsx to understand the concepts behind them. 🙂

Comment (shpomp) [reply react/react1/week2/app/src/components/todosData.jsx:14]: Essentially the files are completely interchangeable, as far as I know. 
It is not that it is super important here, that is why I said it is a nitpick 😅 
It is just clearer if you keep .jsx for components and .js for helpers and data files, but it is not a convention, more like what I have seen, but you could choose your preference 🙏🏽 
I have just through some replies in this https://stackoverflow.com/questions/46169472/reactjs-js-vs-jsx, and it provides some insight.

Comment (RodrigoGuinsbergPinto) [reply react/react1/week2/app/src/components/todosData.jsx:14]: Yes, anyway it was good to read about it and clarify some doubts.
Thanks. 👍🏻

**Pull request merged**